pagopa/io-app
# feat(IT Wallet): [SIW-447] Add credentials issuing info screen

## Short description
This PR adds the info credentials screen that show to the user the credentials requested by the issuer,

## List of changes proposed in this pull request
- Add Credential Issuing Info Screen
- Add a new component to render a grey box with a bullet list of credentials
- Add a new BS to show more info about the issuing flow

NOTE: some data could be mocked

## How to test
Start and activate IT-WALLET, try to add a new credential. It should be possibile to show a screen with a summary of requested credentials
